core/txpool/blobpool/blobpool.go

```diff
@@ -1030,23 +1030,16 @@ func (p *BlobPool) SetGasTip(tip *big.Int) {
 				if tx.execTipCap.Cmp(p.gasTip) < 0 {
 					// Drop the offending transaction
```

Comment:
Pls rephrase `// Drop the offending transaction and everything afterwards, no gaps allowed`